packages/x-charts/src/BarChart/useBarPlotData.ts

```diff
@@ -170,3 +178,28 @@ function getBandSize({
     offset,
   };
 }
+
+function getValueCoordinate(
+  isVertical: boolean,
+  minValueCoord: number,
+  maxValueCoord: number,
+  baseValue: number,
+  minBarSize: number,
+) {
+  const isSizeLessThanMin = Math.abs(maxValueCoord - minValueCoord) < minBarSize;
+  const barSize = isSizeLessThanMin ? minBarSize : Math.abs(maxValueCoord - minValueCoord);
+
+  // Size is above the minimum size, so we can return the bar size and the min value coordinate.
+  if (!isSizeLessThanMin) {
+    return [barSize, minValueCoord];
```

Comment:
Why return an array instead of an object? This isn't a hook nor is it clear the order of the elements. It'd make sense if it were `[x, y]`, for example.